networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/libp2p/PeerManager.java

```diff
@@ -59,7 +62,12 @@ public PeerManager(
     this.reputationManager = reputationManager;
     this.peerHandlers = peerHandlers;
     this.rpcHandlers = rpcHandlers;
-    // TODO - add metrics
+    peerGauge =
+        SettableGauge.create(
+            metricsSystem,
+            ArtemisMetricCategory.NETWORK,
+            "libp2p_peers",
+            "Total number of peers");
```

Comment:
This feels like the wrong approach.  Rather than a settable gauge you could just use the normal gauge and read from `getPeerCount()` on demand.

Reply:
changed to regular gauge